device/bgp/bgproutepathattributes.yaml

```diff
@@ -18,4 +18,10 @@ components:
           enum:
           - igp
           - egp
-          default: igp
\ No newline at end of file
+          default: igp
+        local_preference:
```

Comment:
Might be good to also have a boolean field to exclude it if needed, e.g, include_local_preference , default true ; so that the default ibgp use-case works where this must be present but user has option to exclude from non-default ebgp usecase if desired . It could be made an optional required parameter but that makes is harder for users and also existing scripts need to have this explicitly added else iBGP sessions will flap, so we should definitely have the default value as done now.

Reply:
Agreed, although I would advice against exposing another attribute. Maybe we can have local_preference as an object with one attribute called `value`. It will achieve similar purpose.

e.g. `obj.local_preference.value = 100`